src/course-outline/hooks.jsx

```diff
@@ -187,7 +187,9 @@ const useCourseOutline = ({ courseId }) => {
       dispatch(configureCourseSectionQuery(currentSection.id, ...arg));
       break;
     case COURSE_BLOCK_NAMES.sequential.id:
-      dispatch(configureCourseSubsectionQuery(currentItem.id, currentSection.id, ...arg));
+      dispatch(configureCourseSubsectionQuery(currentItem.id, currentSection.id, ...arg)).then(() => {
+          dispatch(fetchCourseOutlineIndexQuery(courseId));
```

Comment:
This will reload all sections after a subsection is updated. fixes the issue, but I think there could be a more elegant solution